openfl-docker/gramine_app/fx.manifest.template

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# -------------------------------------
+# Enclave Manifest for OpenFL TaskRunner API.
+# This defines the configuration for the Gramine loader to run a Python application.
+# -------------------------------------
+
+libos.entrypoint = "{{ entrypoint }}"
+loader.entrypoint.uri = "file:{{ gramine.libos }}"
+loader.log_level = "{{ log_level }}"
+
+loader.insecure__use_cmdline_argv = true
+loader.insecure__use_host_env = true
+
+loader.env.LD_LIBRARY_PATH = "{{ venv_root }}:{{ arch_libdir }}:/usr/{{ arch_libdir }}:/lib:/usr/lib"
+loader.env.SSL_CERT_DIR = "/etc/ssl/certs"
+
+# Filesystem configuration within Gramine LibOS
+fs.start_dir = "{{ workspace_root }}"
+fs.mounts = [
+  # System mounts (URI: path on host, PATH: pointer inside gramine)
+  { uri = "file:{{ gramine.runtimedir() }}", path = "/lib" },
+  { uri = "file:{{ arch_libdir }}", path = "{{ arch_libdir }}" },
+  { uri = "file:/etc/ssl/certs", path = "/etc/ssl/certs" },
+  { uri = "file:/usr", path = "/usr" },
+  { type = "tmpfs", path = "/tmp" },
+  # User-defined mounts specific to the application.
+  { uri = "file:{{ workspace_root }}", path = "{{ workspace_root }}" },
+  { uri = "file:{{ venv_root }}", path = "{{ venv_root }}" },
+]
+
+# System configuration
+sys.stack.size = "4M"
+sys.brk.max_size = "1M"
+sys.enable_sigterm_injection = true
+sys.enable_extra_runtime_domain_names_conf = true
+
+# SGX configuration
+sgx.debug = false
+sgx.enclave_size = "16G"
+sgx.preheat_enclave = false
+sgx.remote_attestation = "dcap"
+sgx.max_threads = 512
+
+# List of trusted files, that are hashed and signed by the enclave.
+# If these files change after signing of an enclave, application cannot run.
+sgx.trusted_files = [
```

Comment:
I believe the openfl package (incl. `fx` commands) should be explicitly included here

Reply:
Current manifest includes `VENV_ROOT` which contains all python binaries (incl. `fx`) and packages' source code (this is also the approach taken internally).